nucliadb_node/src/services/versions.rs

```diff
@@ -172,10 +172,20 @@ impl Versions {
     }
 
     pub fn load(versions_file: &Path) -> NodeResult<Versions> {
-        let versions_json = std::fs::read_to_string(versions_file)?;
-        let mut versions: Versions = serde_json::from_str(&versions_json)?;
-        versions.fill_gaps();
-        Ok(versions)
+        let deprecated_config = versions_file.parent().map(|v| v.join("config.json"));
```

Comment:
Can you use a constant instead of hardcoding a string?

Reply:
It also appears below

Reply:
Good catch, fix at [ca38165](https://github.com/nuclia/nucliadb/pull/551/commits/ca38165967e7677f5f45a88b4ecc97078cf1546e). Also cleaned a lit bit the struct by defining a function for the duplicated functionality.